server/lib/configuration.js

```diff
@@ -494,6 +501,12 @@ var conf = module.exports = convict({
     },
     sample_rate: 1
   },
+  sync_tokenserver_url: {
+    default: 'http://127.0.0.1:5000/token',
+    doc: 'The url of the Firefox Sync tokenserver',
+    env: 'SYNC_TOKENSERVER_URL',
```

Comment:
I think the general question we need to ask is "what happens when *any* of these config values is wrong or otherwise needs to change?" Let's say that somehow the profile server value was wrong when we query the config - how would we update that for clients who are already signed in and thus will not re-query that config - are they forever stuck with a bad profile server?

To my mind, that question isn't much different from the "what happens if the token server is wrong", even if the likelihoods are different.